app/views/contexts/preview/preview_question.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Union
+
+from flask_babel import lazy_gettext
+from werkzeug.datastructures import ImmutableDict
+
+
+class PreviewQuestion:
+    def __init__(
+        self,
+        *,
+        block: ImmutableDict,
+    ):
+        self._block = block
+        self._block_id = block["id"]
+        self._question = self.get_question()
+        self._title = self._question["title"]
+        self._answers = self._build_answers()
+        self._descriptions = self._question.get("description")
+        self._guidance = self._question.get("guidance")
+        self._type = self._question.get("type")
+
+    def _build_answers(self) -> list[dict]:
+        answers_list = []
+        for answer in self._question.get("answers", []):
+            answer_dict = {}
+            answer_type = answer.get("type")
+            if (answer_label := answer.get("label")) and (answer_type != "Checkbox"):
+                answer_dict["label"] = answer_label
```

Comment:
Should be if options isn't in answer, instead of Checkbox, as it if there is option then we shouldn't need label.

Reply:
Ok, will add this way, I was not sure about it, if something else can have options so I played it safe, since Radio cant have them and Checkbox we don't want to show them.